common/util/proxy.go

```diff
@@ -0,0 +1,153 @@
+package util
```

Comment:
Add a comment of the code is "copy and update from kubernetes/pkg/proxy"

Reply:
done